packages/cli/src/commonlib/cacheAccess.ts

```diff
@@ -191,6 +191,15 @@ export async function saveAccountId(accountName: string, accountId?: string) {
   }
 }
 
+export async function clearCache(accountName: string) {
```

Comment:
Does `msalTokenCache.getAllAccounts()` works for such case?
Similar to https://github.com/OfficeDev/TeamsFx/blob/dev/packages/cli/src/commonlib/codeFlowLogin.ts#L327

Reply:
No, I have tried ```msalTokenCache.removeAccount(account)```, the older version can only remove its own token but not the newer version token.

Reply:
![image](https://user-images.githubusercontent.com/13211513/148171735-cc4a5b14-b184-47e1-bfc5-02e263462f12.png)